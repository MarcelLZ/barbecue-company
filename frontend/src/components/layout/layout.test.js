import React from 'react'
import { shallow } from 'enzyme'
import AppLayout from '.'

describe('<AppLayout />', () => {
  it('should render correctly', () => {
    const component = shallow(<AppLayout />)

    expect(component).toMatchSnapshot()
  })
})
