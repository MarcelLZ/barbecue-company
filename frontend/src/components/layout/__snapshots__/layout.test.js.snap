// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppLayout /> should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AppLayout />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <Header
          title={<span />}
        >
          <Navigation>
            <Link
              replace={false}
              to="/"
            >
              Dashboard
            </Link>
            <Link
              replace={false}
              to="/"
            >
              Pedidos
            </Link>
            <Link
              replace={false}
              to="/"
            >
              Perfil
            </Link>
          </Navigation>
        </Header>,
        <Content />,
      ],
      "fixedHeader": true,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <Navigation>
            <Link
              replace={false}
              to="/"
            >
              Dashboard
            </Link>
            <Link
              replace={false}
              to="/"
            >
              Pedidos
            </Link>
            <Link
              replace={false}
              to="/"
            >
              Perfil
            </Link>
          </Navigation>,
          "title": <span />,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <Link
                replace={false}
                to="/"
              >
                Dashboard
              </Link>,
              <Link
                replace={false}
                to="/"
              >
                Pedidos
              </Link>,
              <Link
                replace={false}
                to="/"
              >
                Perfil
              </Link>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Dashboard",
                "replace": false,
                "to": "/",
              },
              "ref": null,
              "rendered": "Dashboard",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Pedidos",
                "replace": false,
                "to": "/",
              },
              "ref": null,
              "rendered": "Pedidos",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Perfil",
                "replace": false,
                "to": "/",
              },
              "ref": null,
              "rendered": "Perfil",
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <Header
            title={<span />}
          >
            <Navigation>
              <Link
                replace={false}
                to="/"
              >
                Dashboard
              </Link>
              <Link
                replace={false}
                to="/"
              >
                Pedidos
              </Link>
              <Link
                replace={false}
                to="/"
              >
                Perfil
              </Link>
            </Navigation>
          </Header>,
          <Content />,
        ],
        "fixedHeader": true,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <Navigation>
              <Link
                replace={false}
                to="/"
              >
                Dashboard
              </Link>
              <Link
                replace={false}
                to="/"
              >
                Pedidos
              </Link>
              <Link
                replace={false}
                to="/"
              >
                Perfil
              </Link>
            </Navigation>,
            "title": <span />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <Link
                  replace={false}
                  to="/"
                >
                  Dashboard
                </Link>,
                <Link
                  replace={false}
                  to="/"
                >
                  Pedidos
                </Link>,
                <Link
                  replace={false}
                  to="/"
                >
                  Perfil
                </Link>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Dashboard",
                  "replace": false,
                  "to": "/",
                },
                "ref": null,
                "rendered": "Dashboard",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Pedidos",
                  "replace": false,
                  "to": "/",
                },
                "ref": null,
                "rendered": "Pedidos",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Perfil",
                  "replace": false,
                  "to": "/",
                },
                "ref": null,
                "rendered": "Perfil",
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
